### ISSUE TITLE
`minimum` should be a number greater than or equal to `0`

### ISSUE THREAD

**Issue by jradlica** [daveshanley/vacuum#423]
Since version v0.8.0 we encounter error:
```
`minimum` should be a number greater than or equal to `0`
```
for the following field definition:
```
components:
  schemas:
    MyRequest:
      description: desc
      properties:
        numeric-value:
          type: number
          minimum: 0
```

The error suggests that `minimum` can be `equal to 0`, however error occurs for `0` value.

Spec which can be used for reproduction:
```yaml
openapi: 3.0.1
info:
  title: API
  version: 1.0.0
  description: desc
paths:
  /endpoint:
    post:
      operationId: operationX
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/MyRequest'
      responses:
        "200":
          description: Service is up and running.
          content:
            application/json:
              schema:
                type: string
              example:

components:
  schemas:
    MyRequest:
      description: desc
      properties:
        numeric-value:
          type: number
          minimum: 0
```

**Comment by daveshanley** [comment]
will provide a patch to fix this today.

**Comment by daveshanley** [comment]
This is fixed in `v0.8.1`
